mne/commands/tests/test_commands.py

```diff
@@ -177,4 +178,28 @@ def test_surf2bem():
     check_usage(mne_surf2bem)
 
 
+@slow_test
+@requires_freesurfer
```

Comment:
Actually, I suspect Travis will fail on this because `FREESURFER_HOME` is set:

https://github.com/mne-tools/mne-python/blob/master/.travis.yml#L98

This is because we do run some tests with some of the basic freesurfer tools. We just don't have the correct command-line tools for this test, so it should fail. So maybe we do need to have an ultra-slow test switch, and not test those (only this one for now) by default. That will make it so Travis is happy, and so that users won't have their test time doubled. It does mean this function won't get tested often, but that is an okay tradeoff for me for now.